pkg/cmd/browse/browse.go

```diff
@@ -179,6 +179,10 @@ func parseSection(baseRepo ghrepo.Interface, opts *BrowseOptions) (string, error
 		}
 	}
 
+	if opts.SelectorArg == "" {
+		return fmt.Sprintf("tree/%s/", branchName), nil
```

Comment:
Should instead `parseFile()` return an empty `filePath` if the input selector argument was empty to begin with?  Then this explicit condition wouldn't be needed. I see that as the root of the issue.

Reply:
that's a great point! I did not think about it that way. Will do  👍